operator/src/main/java/oracle/kubernetes/operator/helpers/SecretHelper.java

```diff
@@ -62,10 +62,13 @@ public NextAction apply(Packet packet) {
         secretName = dpi.getDomain().getWebLogicCredentialsSecretName();
         namespace = dpi.getNamespace();
 
-        LOGGER.fine(MessageKeys.RETRIEVING_SECRET, secretName);
-        Step read = new CallBuilder().readSecretAsync(secretName, namespace, new SecretResponseStep(getNext()));
-
-        return doNext(read, packet);
+        if (secretName != null) {
```

Comment:
This change is to avoid the NullPointerException when WebLogicCredentialsSecret is removed. I didn't know whether we need unit test when fixing a NPE. I'll look into adding unit test.